mqtt-bridge/src/main/java/io/strimiz/strimzi_mqtt_bridge/Main.java

```diff
@@ -0,0 +1,20 @@
+package io.strimiz.strimzi_mqtt_bridge;
```

Comment:
typo in the first part of the package `io.strimiz`.
I would also use the same pattern we have for the Strimzi HTTP bridge which is `io.strimzi.kafka.bridge.http` so in this case by using `io.strimzi.kafka.bridge.mqtt`. Using underscores in the package names is a bad pattern.

Reply:
Noted sir. I have made the required changes.